accounting/payables/supplier_bills/ocr.rst

```diff
@@ -0,0 +1,39 @@
+=====================================================================
+Digitalize your Vendor Bills with Optical Character Recognition (OCR)
+=====================================================================
+
+Encoding your bills manually can be a time-consuming task. Being able to digitalize them and
+extract the data into your database automatically will allow you to reduce errors, save time and
+have an effortless process.
```

Comment:
Digitize is the right word to use 

I would also remove last sentence it's a repetition 

So will allow you to reduce errors and save time.